x-pack/plugins/security_solution/public/flyout/document_details/left/tabs/insights_tab.tsx

```diff
@@ -103,19 +118,17 @@ export const InsightsTab: React.FC = memo(() => {
     <>
       <EuiButtonGroup
         color="primary"
-        name="coarsness"
         legend={i18n.translate(
           'xpack.securitySolution.flyout.left.insights.buttonGroupLegendLabel',
-          {
-            defaultMessage: 'Insights options',
-          }
+          { defaultMessage: 'Insights options' }
         )}
-        options={insightsButtons}
+        options={buttonGroup}
         idSelected={activeInsightsId}
         onChange={onChangeCompressed}
         buttonSize="compressed"
         isFullWidth
         data-test-subj={INSIGHTS_TAB_BUTTON_GROUP_TEST_ID}
+        style={!isEventKindSignal ? { maxWidth: 300 } : undefined}
```

Comment:
is that change wanted? I don't remember seeing it in my last review. Is that the reason the UI looks like this?
![Screenshot 2024-02-15 at 4 19 43 PM](https://github.com/elastic/kibana/assets/17276605/3da5f273-2062-41a3-89fc-4957e9388b87)
I feel like it's a bit weird tbh, it's even smaller than when all the tabs and sub tabs are present

Reply:
yep, it's the updated mock from design, I will tag you in the thread